### DISCUSSION TITLE
MCprep Russian Translation (ru_RU)

### DISCUSSION THREAD

**Discussion by StandingPadAnimations** [Moo-Ack-Productions/MCprep#579] [Translation Peer Review]
This is the peer review thread for the Russian Translation (ru_RU) of MCprep. 

Pull Request: #578 

Download of build (no assets):  [MCprep_addon.zip](https://github.com/Moo-Ack-Productions/MCprep/files/15311928/MCprep_addon.zip)

Some screenshots:
![image](https://github.com/Moo-Ack-Productions/MCprep/assets/75058058/ae214b25-0aaa-4501-868e-ebb93b0c0833)

![image](https://github.com/Moo-Ack-Productions/MCprep/assets/75058058/0da3bb54-7dcc-40d9-953e-328fedd68b37)

**Comment by TheDuckCow** [comment]
Just [cross posted here](https://www.youtube.com/post/Ugkx39pjHaZzVDf9e7MshcIVtxjJDC1S8bgI)

**Comment by Spacy80893** [comment]
legit
